avocado_vt/plugins/vt_joblock.py

```diff
@@ -86,35 +87,17 @@ def _get_lock_files(self):
             if e.errno == errno.ENOENT:
                 return []
 
-    def _get_lock_file_pid(self):
-        """
-        Gets the lock file path and the process ID
-
-        If the lock file can not be located, this returns (None, 0).
-
-        :returns: the path and the (integer) process id
-        :rtype: tuple(str, int)
-        """
-        files = self._get_lock_files()
-        if not files:
-            return (None, 0)
-        path = files[0]
-        content = int(open(path, 'r').read())
-        pid = int(content)
-        if pid > 0:
-            return (path, pid)
-
     def _lock(self, job):
-        filename, lock_pid = self._get_lock_file_pid()
-        if lock_pid > 0:
-            msg = ('Avocado-VT job lock file "%s" acquired by PID %u. '
-                   'Aborting...' % (filename, lock_pid))
-            self._abort(msg, job)
         self.lock_file = self._create_self_lock_file(job)
-
-    def _unlock(self):
-        if self.lock_file:
-            os.unlink(self.lock_file)
+        lock_files = self._get_lock_files()
+        lock_files.remove(self.lock_file)
+        for path in lock_files:
+            lock_pid = int(open(path, 'r').read())
```

Comment:
I'd suggest doing this in `try/except` block as there might be corrupted files or files which are created but does not have the `pid` written yet.

In that case I'd suggest aborting the job, because we don't know whether it's just a random file, or whether other process just started.